### ISSUE TITLE
OVAL validation fails for ism_o

### ISSUE THREAD

**Issue by comps** [ComplianceAsCode/content#9828]
#### Description of problem:

```
oscap oval validate --results --schematron /var/lib/libvirt/images/ism_o/ssg-rhel8-oval.xml.result.xml
```
seems to fail on
```
File '/var/lib/libvirt/images/ism_o/ssg-rhel8-oval.xml.result.xml' line 65535: Element '{http://oval.mitre.org/XMLSchema/oval-system-characteristics-5#independent}value_of', attribute 'datatype': [facet 'enumeration'] The value '' is not an element of the set {'binary', 'boolean', 'evr_string', 'fileset_revision', 'float', 'ios_version', 'int', 'ipv4_address', 'ipv6_address', 'string', 'version'}.
File '/var/lib/libvirt/images/ism_o/ssg-rhel8-oval.xml.result.xml' line 65535: Element '{http://oval.mitre.org/XMLSchema/oval-system-characteristics-5#independent}value_of', attribute 'datatype': '' is not a valid value of the atomic type '{http://oval.mitre.org/XMLSchema/oval-common-5}SimpleDatatypeEnumeration'.
OpenSCAP Error: Invalid OVAL Results (5.11) content in /var/lib/libvirt/images/ism_o/ssg-rhel8-oval.xml.result.xml. [/builddir/build/BUILD/openscap-1.3.6/src/source/oscap_source.c:353]
```

#### SCAP Security Guide Version:
master as of 2022-11-15

#### Operating System Version:
RHEL-9.1 and RHEL-8.7 internal nightlies

#### Additional Information/Debugging Steps:
Found via `Sanity/ssg-kickstarts for ISM`. I've retried running the test 4 times in total, so this is likely not an infrastructure/random issue.

**Comment by jan-cerny** [comment]
This is an issue in OpenSCAP. OpenSCAP produces an invalid XML. It is already tracked in https://github.com/OpenSCAP/openscap/issues/1890 and https://bugzilla.redhat.com/show_bug.cgi?id=2138884

**Comment by mildas** [comment]
Blocked on openscap scanner

**Comment by vojtapolasek** [comment]
Just a note that this behavior manifested also with STIG profile on RHEL9 when running Automatus after kickstart installation.

**Comment by mildas** [comment]
The issue was fixed in openscap component - https://bugzilla.redhat.com/show_bug.cgi?id=2139060 and https://bugzilla.redhat.com/show_bug.cgi?id=2138884
